integration-tests/features/stack_analyses_npm_ecosystem.feature

```diff
@@ -101,14 +101,14 @@ Feature: Thorough stack analysis v3 API tests - NPM ecosystem
     Then I should find at least 2 dependencies
      And I should find the following dependencies (base64-encode, commander) in the stack analysis
      And I should find dependency named base64-encode with version 1.0.1 in the stack analysis
-     And I should find dependency named commander with version 2.20.0 in the stack analysis
```

Comment:
i am sure these tests are for the static files. why is there a need to change the versions?

Reply:
Stack analysis was returning a different version , so changed it to the version returned by stack analysis

Reply:
I then see its a problem with the input file. In the input, we specify the pkg along with version. If the input has pkgs without version specified, then please change it

Reply:
`"commander": ">=2.2.0"` see the input file is sending version like this because of the greater than symbol it is returning a greater version i.e `5.1.0`